src/compas_fab/robots/wrench.py

```diff
@@ -0,0 +1,353 @@
+		return '''Wrench(Fx {0:.2f}, Fy {1:.2f}, Fz {2:.2f}, Tx {3:.2f}, Ty {4:.2f}, Tz {5:.2f})'''.format(*self.wrench)
+
+	# ==========================================================================
+	# helpers
+	# ==========================================================================
+
+	def copy(self):
+		"""Make a copy of this ``Wrench``.
+
+		Returns
+		-------
+		Wrench
+			The copy.
+		"""
+		cls = type(self)
+		return cls(copy.copy(self.wrench))
+
+	# ==========================================================================
+    # methods
+    # ==========================================================================
+	
+	@staticmethod
+	def binary(wrench, forces_threshold = 10.0, torques_threshold = 5.0):
+		"""
+		Discretize the wrench
+		"""
+		binary_ft = [0,0,0,0,0,0]
+
+		#forces
+		for idx, w in enumerate(wrench[0:3]):
+			if (w > forces_threshold): binary_ft[idx] = 1
+			elif (w < -forces_threshold): binary_ft[idx] = -1
+			else: binary_ft[idx] = 0
+		#torques
+		for idx, w in enumerate(wrench[3:6]):
+			if (w > torques_threshold): binary_ft[idx+3] = 1
+			elif (w < -torques_threshold): binary_ft[idx+3] = -1
+			else: binary_ft[idx+3] = 0
+		
+		return Wrench(binary_ft)
+	
+	# ==========================================================================
+	# operators
+	# ==========================================================================
+
+	def __mul__(self, n):
+		"""Create a ``Wrench`` multiplied by the given factor.
+
+		Parameters
+		----------
+		n : float
+			The multiplication factor.
+
+		Returns
+		-------
+		Wrench
+			The resulting new ``Wrench``.
+
+		"""
+		return Wrench([
+			n * self.wrench[0],
+			n * self.wrench[1],
+			n * self.wrench[2],
+			n * self.wrench[3],
+			n * self.wrench[4],
+			n * self.wrench[5] ])
+	
+	def __sub__(self, other):
+		"""Return a ``Wrench`` that is the the difference between this ``Wrench`` and another wrench.
+
+		Parameters
+		----------
+		other : wrench
+			The wrench to subtract.
+
+		Returns
+		-------
+		Wrench
+			The resulting new ``Wrench``.
+		"""
+		return Wrench([
+			self.wrench[0] - other[0],
+			self.wrench[1] - other[1],
+			self.wrench[2] - other[2],
+			self.wrench[3] - other[3],
+			self.wrench[4] - other[4],
+			self.wrench[5] - other[5] ])
+	
+	def __neg__(self):
+		"""Return the negated ``Wrench``.
+
+		Returns
+		-------
+		Wrench
+			The negated ``Wrench``.
+		"""
+		return Wrench([
+			-1.0 * self.wrench[0],
+			-1.0 * self.wrench[1],
+			-1.0 * self.wrench[2],
+			-1.0 * self.wrench[3],
+			-1.0 * self.wrench[4],
+			-1.0 * self.wrench[5] ])
+
+	def __len__(self):
+		return len(self.wrench)
+	
+	def __getitem__(self, item):
+		return self.wrench[item]
+	
+	def __eq__(self, other):
+		return np.allclose(self.wrench, other)
+
+	# ==========================================================================
+	# transformations
+	# ==========================================================================
+
+	def rotated(self, rotation_matrix):
```

Comment:
I tried what you suggested but I had some errors. So instead of using the object Transformation, I just used a matrix, as in the compas geometry Vector.transform(matrix)